qiskit_experiments/framework/experiment_data.py

```diff
@@ -652,6 +652,19 @@ def _set_hgp_from_provider(self, provider):
         except (AttributeError, IndexError):
             return
 
+    @property
+    def hgp(self) -> str:
+        """Returns Hub/Group/Project data as a formatted string"""
+        return f"{self.hub}/{self.group}/{self.project}"
+
+    @hgp.setter
+    def hgp(self, new_hgp: str) -> None:
+        """Sets the Hub/Group/Project data from a formatted string"""
+        hub, group, project = new_hgp.split("/")
+        self._db_data.hub = self._db_data.hub or hub
```

Comment:
I don't see a scenario where `hub` is None

Reply:
Agree. Changed this.